database/schema.ts

```diff
@@ -292,6 +292,38 @@ export const purchasesTable = mysqlTable('purchases', {
     .notNull(),
 });
 
+export const bannerJobTable = mysqlTable('banner_jobs', {
+  id: bigint('id', {
+    unsigned: true,
+    mode: 'number',
+  })
+    .autoincrement()
+    .primaryKey(),
+  user_id: mediumint('user_id', {
```

Comment:
```suggestion
  userId: mediumint('user_id', {
```

would really prefer it like this but I haven't really kept to this either especially early on so dw about it too much